dataclasses_json/cfg.py

```diff
@@ -50,7 +50,7 @@ def config(metadata: dict = None, *,
            letter_case: Callable[[str], str] = None,
            undefined: Optional[Union[str, Undefined]] = None,
            field_name: str = None,
-           exclude: Optional[Callable[[str, T], bool]] = None,
+           exclude: Optional[Callable[[T], bool]] = None,
```

Comment:
Should the function input be a `core.Json`, instead of an arbitrary `T`?

```suggestion
           exclude: Optional[Callable[[Json], bool]] = None,
```

Reply:
`from core import Json`

Reply:
We do convert `tuple` and `set` to `list`, and nested dataclasses to `dict`, before passing them to `exclude`. However, we pass `datetime`, `UUID`, and `Decimal` objects directly to `exclude` as-is. (Is that discrepancy intended?)